[2024-07-04 20:31:59,917] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:31:59] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:32:53,522] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:32:53] "GET /busca_pacientes HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:32:58,032] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:32:58] "GET /busca_pacientes HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:33:03,535] WARNING add_paciente() L56   Erro ao adicionar paciente 'teste', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:33:03,536] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:33:03] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:33:38,852] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:33:38] "GET /busca_pacientes HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:33:57,127] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:33:57] "GET /busca_pacientes HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:34:14,406] WARNING add_paciente() L56   Erro ao adicionar paciente 'bererer', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:34:14,406] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:34:14] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:39:25,440] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:39:25] "POST /adiciona_paciente HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:42:42,824] WARNING add_paciente() L56   Erro ao adicionar paciente 'bererer', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:42:42,826] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:42:42] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:43:57,748] WARNING add_paciente() L56   Erro ao adicionar paciente 'rrutyuty', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:43:57,750] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:43:57] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:45:47,076] WARNING add_paciente() L56   Erro ao adicionar paciente 'ddddddd', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:45:47,077] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:45:47] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:48:37,951] WARNING add_paciente() L56   Erro ao adicionar paciente 'trtrtrtrt', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:48:37,954] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:48:37] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:50:54,490] WARNING add_paciente() L56   Erro ao adicionar paciente 'yuyuyu', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:50:54,490] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:50:54] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:51:56,275] WARNING add_paciente() L56   Erro ao adicionar paciente 'yuyuyu', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:51:56,276] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:51:56] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 20:52:04,782] WARNING add_paciente() L56   Erro ao adicionar paciente 'trtrtr', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L56  
[2024-07-04 20:52:04,783] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 20:52:04] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:20:12,291] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.23:5000 - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:20:12,296] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:20:33,283] WARNING add_paciente() L58   Erro ao adicionar paciente 'ddddddd', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L58  
[2024-07-04 21:20:33,283] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:20:33] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:21:48,386] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:21:48] "OPTIONS /adiciona_paciente HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:21:48,436] WARNING add_paciente() L58   Erro ao adicionar paciente 'JOSERRR', Paciente de mesmo nome já salvo na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L58  
[2024-07-04 21:21:48,454] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:21:48] "[31m[1mPOST /adiciona_paciente HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:29:35,305] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:29:35] "OPTIONS /adiciona_atendimento HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:29:35,328] WARNING add_atendimento() L184  Erro ao adicionar atendimento ao paciente '1', Paciente já está na fila de atendimento :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L184 
[2024-07-04 21:29:35,329] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:29:35] "[31m[1mPOST /adiciona_atendimento HTTP/1.1[0m" 409 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:04,297] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:04] "DELETE /remove_paciente?id=1 HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:08,700] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:08] "DELETE /remove_paciente?id=3 HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:11,703] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:11] "DELETE /remove_paciente?id=2 HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:14,338] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:14] "DELETE /remove_paciente?id=4 HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:27,429] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:27] "OPTIONS /adiciona_atendimento HTTP/1.1" 200 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-07-04 21:33:27,440] WARNING add_atendimento() L160  Erro ao adicionar atendimentos ao paciente '1', Paciente não encontrado na base :/ - call_trace=C:\projetospos\MVP-Sprint2\Terminal_Triagem_Backend\app.py L160 
[2024-07-04 21:33:27,443] INFO _log() L96   127.0.0.1 - - [04/Jul/2024 21:33:27] "[33mPOST /adiciona_atendimento HTTP/1.1[0m" 404 - - call_trace=C:\projetospos\MVP-Sprint1\Terminal_Triagem_Backend\env\Lib\site-packages\werkzeug\_internal.py L96  
